telegram/commands/track/wishlist.go

```diff
@@ -0,0 +1,82 @@
+package track
```

Comment:
i think this feature it's a command itself, this is not an activity so it shouldnt be in the track folder